vmdk_plugin/sanity_test.go

```diff
@@ -250,6 +253,9 @@ func TestSanity(t *testing.T) {
 				volumeName, elem.endPoint)
 		}
 	}
+	fmt.Printf("%s END: Running TestSanity on  %s (may take a while)...\n",
```

Comment:
True but I am trying to be consistent with START block at L213 'fmt.Printf("%s START:'